debugger/src/main/kotlin/org/rust/debugger/settings/RsStepFilterSettings.kt

```diff
@@ -0,0 +1,10 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.debugger.settings
+
+data class RsStepFilterSettings(
```

Comment:
@Kobzol I guess that `XmlSerializerUtil` does not properly serialize a Kotlin data class out-of-the-box. So it's probably worth just using a primitive `var skipStdlib: Boolean = false` field in the `RsDebuggerSettings` instead of boxing it into `data class RsStepFilterSettings`. Just the same way as we do with `var breakOnPanic: Boolean = true`